aeron-driver/src/main/java/uk/co/real_logic/aeron/driver/DriverConductor.java

```diff
@@ -683,6 +684,11 @@ private void onRemoveSubscription(final long registrationId, final long correlat
         final int refCount = channelEndpoint.decRefToStream(subscription.streamId());
         if (0 == refCount)
         {
+            connections
```

Comment:
When the Receiver does a removeSubscription from the dispatcher, I think it does this check already. Why do it here?